sway-lib-std/src/math.sw

```diff
@@ -51,6 +51,36 @@ pub trait Power {
     fn pow(self, exponent: Self) -> Self;
 }
 
+
+impl Power for u256 {
+    /// Raises self to the power of `exp`, using exponentiation by squaring.
+	///
+	/// # Panics
+	///
+	/// Panics if the result overflows the type.
+    fn pow(self, exp: Self) -> Self {
```

Comment:
This is from the original trait, but exp should be a `u32` or a `u64` but definitely not a `Self`.

Reply:
I can make this change now. What do you think?
Will be a breaking change.

Reply:
We should probably do this in a separate PR.